tests/scripts/conformance_e2e.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright 2024 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+KUBEEDGE_ROOT=$PWD
+TEST_DIR=$(realpath $(dirname $0)/..)
+
+GOPATH=${GOPATH:-$(go env GOPATH)}
+KIND_IMAGE=${1:-"kindest/node:v1.26.0"}
```

Comment:
check main.yaml , kindest version is v1.28.0 . here kindest version is v1.26.0.  Should it use the same version?

Reply:
done